sources/metadata/README.md

```diff
@@ -0,0 +1 @@
+This directory holds source files for the `metadata` endpoint split by products.
```

Comment:
Should we add the real metadata for Everest 0.8.0 in this PR?

Reply:
It can be added a bit later. There's a process for dev/prod environments.